packages/core/src/app/formFields/getCustomFormFieldsValidationSchema.ts

```diff
@@ -29,7 +29,7 @@ export interface CustomFormFields {
 
 export default memoize(function getCustomFormFieldsValidationSchema({
     formFields,
-    translate = () => undefined,
+    translate = () => '',
```

Comment:
Think we looked at this and agreed to keep it as `undefined` ?

Reply:
yep, will change it back to undefined